sound/soc/sof/intel/hda.h

```diff
@@ -447,6 +447,9 @@ struct sof_intel_hda_dev {
 
 	/* sdw context allocated by SoundWire driver */
 	struct sdw_intel_ctx *sdw;
+
+	/* FW clock config, 0:LPRO, 1:HPRO */
```

Comment:
this comment seems wrong isnt it? It will actually be 1 for LPRO and 0 for HPRO right?

Reply:
I'd suggest we add enums to be more descriptive CAVS_CLK_CONFIG_LPRO , CAVS_CLK_CoNFIG_HPRO and set clk_config to the appropriate value depending on whether the FW is configured for LPRO or not

Reply:
oops, that's right. 1 for LPRO and 0 for HPRO